src/migrations/migrate.rs

```diff
@@ -176,6 +185,13 @@ pub async fn migrate(cli: &mut Connection, _options: &Options,
         }
     }
     cli.execute("COMMIT").await?;
+
+    cli.execute(format!(
```

Comment:
This is not a correct way to undo timeout, since it will not restore the timeout when commit fails but connection is still alive.

Reply:
`migration create` has similar code.  Can you suggest a better way?

Reply:
In what scenario would connection be still alive here?  I made `COMMIT` fail and the whole thing crashed.

Reply:
Not exactly. It doesn't jump out of here (no question mark):
https://github.com/edgedb/edgedb-cli/blob/fd3307282eb1fafdb65fed954d0694665d8c3965/src/migrations/create.rs#L553

And then carefully ensures that all statements have been executed regardless of the results (and errors if any are propagated only after that):
https://github.com/edgedb/edgedb-cli/blob/fd3307282eb1fafdb65fed954d0694665d8c3965/src/migrations/create.rs#L557-L563

Although, these, have probably slipped into the code by a mistake:
https://github.com/edgedb/edgedb-cli/blob/fd3307282eb1fafdb65fed954d0694665d8c3965/src/migrations/create.rs#L530-L533

So yes, that has to be fixed too.

Anyway, I can take care of all of that.